newsfragments/653.feature

```diff
@@ -0,0 +1 @@
+``FormatROD``: include support for multi-axis goniometers and faster decompression.
```

Comment:
No, only FormatROD

Reply:
I think what we can say is that instead of partial support it is "complete support, with fast decompression" @biochem-fan is that a reasonable release note?

Reply:
I would avoid "complete" because it does not support older (non-default) compression schemes and _very_ distorted geometries (there are correction fields in the header but I don't know how to interpret them).